src/react/queries.ts

```diff
@@ -276,3 +277,20 @@ export const getObjectsVersions = ({
         }
       : undefined,
 });
+
+export const getVeeamObject = ({
+  bucketName,
+  s3Client,
+  veeamKey = `${VEEAM_XML_PREFIX}/capacity.xml`,
+}: {
+  bucketName: string;
+  s3Client: S3;
+  veeamKey?: string;
+}) => ({
+  queryKey: ['getVeeamObject', bucketName],
+  queryFn: () => {
+    return s3Client.getObject({ Bucket: bucketName, Key: veeamKey }).promise();
+  },
+  refetchOnMount: false,
+  refetchOnWindowFocus: false,
+});
```

Comment:
nit: I think we could just have a generic getObjectQuery so that this code could be reused outside of veeam specific context